docs/doc/10-deploy/06-manage-settings.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Managing Settings
+sidebar_label: Managing Settings
+description:
+  Managing Databend settings
+---
+
+Databend provides a variety of system settings that enable you to control how Databend works. For example, you can set a timezone in which Databend works and the SQL dialect you prefer. 
+
+System settings in Databend are categorized into two levels: **Session** and **Global**. A session-level setting applies to the current node only, while a global-level setting affects the entire cluster. A session-level setting can be converted to a global-level one, and it's also possible the other way around.
```

Comment:
Global setting affects all cluster of one tenant. session setting affects one connection. CC: @soyeric128